### PR TITLE
[PT-BR] Translating playground-examples files

### PR DESCRIPTION
**Pull request by Perkles**
This current PR translate the following files: 
 - [JSDoc Deprecated.ts](https://github.com/microsoft/TypeScript-Website/blob/v2/packages//playground-examples/copy/en/4-0/New%20JS%20Features/JSDoc%20Deprecated.ts)
 - [Logical Operators and Assignment.ts](https://github.com/microsoft/TypeScript-Website/blob/v2/packages//playground-examples/copy/en/4-0/New%20JS%20Features/Logical%20Operators%20and%20Assignment.ts) 
 - [Nullish Coalescing.ts](https://github.com/microsoft/TypeScript-Website/blob/v2/packages//playground-examples/copy/en/4-0/New%20JS%20Features/Nullish%20Coalescing.ts)

Those new translated files will be located at `packages/playground-examples/copy/pt/4-0/New JS Features/...`. 

Ref: #233 

@khaosdoctor, @alvarocamillont, @danilofuchs e @orta

### CODE REVIEW

**Comment by khaosdoctor** [comment]
Hi @Perkles! Since your repo is on WIP, can you please convert the PR to draft?

Thanks!

File: packages/playground-examples/copy/pt/4-0/New JS Features/JSDoc Deprecated.ts [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+  /** @deprecated use o campo de gênero no lugar */
+  sex: "male" | "female";
+}
+
+declare const userInfo: AccountInfo;
+userInfo.sex;
+
+// TypeScript irá oferecer um aviso não bloqueante quando 
+// uma propriedade depreciada for acessada, e editores como
```
Comment (danilofuchs) [inline packages/playground-examples/copy/pt/4-0/New JS Features/JSDoc Deprecated.ts]:
```suggestion
// uma propriedade descontinuada for acessada, e editores como
```

File: packages/playground-examples/copy/pt/4-0/New JS Features/Logical Operators and Assignment.ts [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+// podendo poupar uma boa quantidade de código repetido.
+
+declare const user: User
+user.location.postalCode ||= "90210"
+
+// São três novos operadores: 
+//
+//   ||= mostrado acima
+//   &&= que usa a lógica do 'and' ao invés da 'or'
+//   ??= que se baseia no exemplo:nullish-coalescing para oferecer
```
Comment (danilofuchs) [inline packages/playground-examples/copy/pt/4-0/New JS Features/Logical Operators and Assignment.ts]:
```suggestion
//   ??= que se baseia no example:nullish-coalescing para oferecer
```
`example:*` é um link

File: packages/playground-examples/copy/pt/4-0/New JS Features/Nullish Coalescing.ts [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+
+const undefinedValue = response.undefinedValue || "some other default";
+// Seria: 'some other default'
+
+const nullValue = response.nullValue || "some other default";
+
+// Esses dois exemplos funcionam de maneira similar na maioria
+// das linguagens. Como ferramenta || é muito bom em padronizar coisas
+// mas o falsy checks do Javascript pode te surpreender com alguns 
+// valores simples:
```
Comment (danilofuchs) [inline packages/playground-examples/copy/pt/4-0/New JS Features/Nullish Coalescing.ts]:
```suggestion

// Esses dois exemplos funcionam de maneira similar na maioria
// das linguagens. A ferramenta || é muito boa em padronizar coisas
// mas as checagens de falsidade do Javascript podem te surpreender 
// com alguns valores simples:
```

**Review by danilofuchs — CHANGES_REQUESTED** [review]

**Comment by danilofuchs** [comment]
LGTM! Thank you for your effort!

**Comment by khaosdoctor** [comment]
@perkles please update the spreadsheet on #233 so we know you have finished :)

**Comment by Perkles** [comment]
Sure. I did it a couple days ago when the PR was merged. We are talking about [this](https://docs.google.com/spreadsheets/d/14LtRy1HB9n8vy_8JvVu_Pk-fG359v64Knp_3lWJz0cg/edit#gid=0) spreadsheet, right ?

**Pull request merged**